discord/client.py

```diff
@@ -888,9 +888,15 @@ def get_all_channels(self) -> Generator[GuildChannel, None, None]:
         for guild in self.guilds:
             yield from guild.channels
 
-    def get_all_members(self) -> Generator[Member, None, None]:
+    def get_all_members(self, include_bots: Optional[bool]=True) -> Generator[Member, None, None]:
```

Comment:
why not make it a kwarg?
```suggestion
    def get_all_members(self, *, include_bots: Optional[bool]=True) -> Generator[Member, None, None]:
```